x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/engines_table.test.tsx

```diff
@@ -140,4 +172,75 @@ describe('EnginesTable', () => {
       expect(tableContent).not.toContain('Universal');
     });
   });
+
+  describe('actions', () => {
+    it('when AppLogic.values.myRole.canManageEngines is false actions are hidden', () => {
+      jest.clearAllMocks();
+      setMockValues({
+        // AppLogic
+        myRole: {
+          canManageEngines: false,
+        },
+      });
+      const wrapper = mountWithIntl(<EnginesTable {...props} />);
```

Comment:
Because you're no longer using `FormattedMessage`, you actually don't need to use `mountWithIntl`

Reply:
0f879eb9604e780e066c1afc5bb072c1dee4cfc8